This support script ends up hardcoding unnecessary rpaths into the libraries. We
will search $libdir automatically so this is just wastes space. There may be some
cases this is necessary but our use cases aren't one of them.

Upstream-Status: Inappropriate

RP 2012/2/23

Rebase to 8.2.13
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 support/shobj-conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/support/shobj-conf b/support/shobj-conf
index cd7634d..029123a 100644
--- a/support/shobj-conf
+++ b/support/shobj-conf
@@ -128,7 +128,7 @@ linux*-*|gnu*-*|k*bsd*-gnu-*|midnightbsd*|freebsd*|dragonfly*)
 	SHOBJ_LD='${CC}'
 	SHOBJ_LDFLAGS='-shared -Wl,-soname,$@'
 
-	SHLIB_XLDFLAGS='-Wl,-rpath,$(libdir) -Wl,-soname,`basename $@ $(SHLIB_MINOR)`'
+	SHLIB_XLDFLAGS='-Wl,-soname,`basename $@ $(SHLIB_MINOR)`'
 	SHLIB_LIBVERSION='$(SHLIB_LIBSUFF).$(SHLIB_MAJOR)$(SHLIB_MINOR)'
 	;;
 
-- 
2.27.0

